tests/snapshots/tests.contrib.azure_functions.test_azure_functions_snapshot.test_http_get_ok.json

```diff
@@ -0,0 +1,35 @@
+[[
+  {
+    "name": "azure.functions.invoke",
+    "service": "azure_functions",
+    "resource": "GET /api/httpgetok",
+    "trace_id": 0,
+    "span_id": 1,
+    "parent_id": 0,
+    "type": "serverless",
+    "error": 0,
+    "meta": {
+      "_dd.base_service": "",
```

Comment:
Just leaving a comment to check on where we want base service to be blank, though @wconti27 may have the answer.